src/io.jl

```diff
@@ -53,111 +79,33 @@ function create_parameters_and_sets_from_file(input_folder::AbstractString)
 
     # Parameter for profile of assets
     assets_profile = Dict(
-        (assets[row.id], row.rep_period_id, row.time_step) => row.value for
+        (asset_data[row.id][1], row.rep_period_id, row.time_step) => row.value for
         row in eachrow(assets_profiles_df)
     ) # asset profile [p.u.]
 
     # Parameter for profile of flow
     flows_profile = Dict(
-        (flows[row.id], row.rep_period_id, row.time_step) => row.value for
+        (flow_data[row.id][1], row.rep_period_id, row.time_step) => row.value for
```

Comment:
Unlike the first one, there is no great way to do this. The simplest way is to create a vector of `edges` relating ths id to the "names" of the edges. This will create an unnecessary array, because the information is already available in `flow_data`.
It looks like the correct way to do it would be to change the input data and remove the id and use `from` and `to` as indexes.